kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/JRF-AI/mii-update1-d1-JRF-AI-v1-ds.yaml

```diff
@@ -147,8 +133,10 @@ spec:
     model:
       # Valid model domain types are 'WLS', 'JRF', and 'RestrictedJRF', default is 'WLS'
       domainType: "JRF"
-      modelHome: "/auxiliary/models"
```

Comment:
IMO, since this sample is specific to auxiliary image, adding "modelHome" and "wdtInstallHome" might be confusing since the aux image model and install copies always go to default location. If the user sets "modelHome" or "wdtInstallHome" to some other value than the default, we ignore the copies from the auxiliary image. We can document this and make it clear there but I'm not sure if adding them back to auxiliary image specific sample will be helpful or not.